static/scripts/topicEdit.js

```diff
@@ -660,7 +701,8 @@ class TopicGeoGebra extends TopicBlock {
                         title={$t('topic.topicEdit.label.youllFindTheIdOn')}><i className="fa fa-info-circle" /></a>
                 </span>
                 <input
```

Comment:
Why don't we have a visual label here? It is always better to have a visual label that will benefit non-visually-impaired users too. Aria-label benefits only screen reader users.